src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/LightBulbHelper.cs

```diff
@@ -62,7 +62,7 @@ public static async Task<IEnumerable<SuggestedActionSet>> WaitForItemsAsync(Test
                 if (e.Status == QuerySuggestedActionCompletionStatus.Completed)
                     tcs.SetResult(e.ActionSets.ToList());
                 else
-                    tcs.SetException(new InvalidOperationException($"Light bulb transitioned to non-complete state: {e.Status}"));
+                    tcs.TrySetException(new InvalidOperationException($"Light bulb transitioned to non-complete state: {e.Status}"));
```

Comment:
📝 I observed this in an integration test run recently. It races with TrySetCanceled for unexpected dismissal.